tests/client/notifications_api/test_direct_notification.py

```diff
@@ -955,3 +956,115 @@ async def test__send_message__all_mentions_types_succeed(
     # - Assert -
     assert (await task) == UUID("21a9ec9e-f21f-4406-ac44-1a78d2ccf9e3")
     assert endpoint.called
+
+
+@respx.mock
+@pytest.mark.asyncio
+@pytest.mark.mock_authorization
+async def test__send_message__message_body_max_length_success(
+    httpx_client: httpx.AsyncClient,
+    host: str,
+    bot_id: UUID,
+    bot_account: BotAccount,
+) -> None:
+    max_long_body = "1" * constants.MAXIMUM_TEXT_LENGTH
```

Comment:
Don't use constant here, use value. Otherwise, tests will not show a value change.

Reply:
Fix.